### PR TITLE
top holders: add pagination for topHolders method 144

### PR DESCRIPTION
**Pull request by lzxm160**
work on #144 

query topholders(
$endEpochNumber: Int!
$pagination:Pagination!
)
{
 topHolders(endEpochNumber:$endEpochNumber,pagination:$pagination)
{
  address
  balance         
}
}
query variables:
{
  "endEpochNumber":100,
 "pagination":{
"first":10,
"skip":0
}
}

or：
query topholders
{
 topHolders(endEpochNumber:100,pagination:{first:10,skip:0})
{
  address
  balance         
}
}

### CODE REVIEW

File: queryprotocol/actions/protocol.go [hunk 1]
```diff
@@ -482,15 +482,12 @@ func (p *Protocol) GetXrc20ByPage(numPerPage, page uint64) (cons []*Xrc20Info, e
 }
 
 // GetTopHolders gets top holders
-func (p *Protocol) GetTopHolders(endEpochNumber, numberOfHolders uint64) (holders []*TopHolder, err error) {
+func (p *Protocol) GetTopHolders(endEpochNumber, skip, first uint64) (holders []*TopHolder, rr error) {
```
Comment (lizhefeng) [inline queryprotocol/actions/protocol.go]: Why rr?

File: queryprotocol/actions/protocol.go [hunk 2]
```diff
@@ -482,15 +482,12 @@ func (p *Protocol) GetXrc20ByPage(numPerPage, page uint64) (cons []*Xrc20Info, e
...
 	if _, ok := p.indexer.Registry.Find(actions.ProtocolID); !ok {
 		return nil, errors.New("actions protocol is unregistered")
 	}
 	db := p.indexer.Store.GetDB()
-	if numberOfHolders < 1 {
-		numberOfHolders = 1
-	}
-	getQuery := fmt.Sprintf(selectAccountIncome, accounts.AccountIncomeTableName, endEpochNumber, numberOfHolders)
+	getQuery := fmt.Sprintf(selectAccountIncome, accounts.AccountIncomeTableName, endEpochNumber, skip, first)
```
Comment (lizhefeng) [inline queryprotocol/actions/protocol.go]: We also need to check skip and first are valid parameters.

Comment (lzxm160) [reply queryprotocol/actions/protocol.go]: sure

**Comment by lizhefeng** [comment]
Please modify the error message and use the maximum size limit based on this commit: https://github.com/iotexproject/iotex-analytics/commit/2faa6b74cecd12d960c3ab5ecdc0a02a7193f51b

**Pull request merged**